packages/web-components/fast-foundation/src/utilities/template-helpers.ts

```diff
@@ -11,6 +11,7 @@ import type { SyntheticViewTemplate } from "@microsoft/fast-element";
  * @remarks
  * When providing a string, take care to ensure that it is
  * safe and will not enable an XSS attack.
+ * @public
```

Comment:
This was non-public because I wasn't sure about committing to this as a supported API.

Reply:
Should I tag it beta then? The following function `staticallyCompose` will need to match, but it's currently public.

Reply:
Hmm. It's ok as is I think. @chrisdholt may have already changed it at some point to be public. It's probably harmless since it's such a simple little utility function.

Reply:
`staticallyCompose` is public, which makes API Explorer throw a warning because it takes a parameter of `StaticallyComposableHTML` which has no release tag. Since the function was public I made the type public. Since we've already made the function public I don't see a downside of making the type public.

The whole package is technically in beta right now anyway.